content/apps/deployment.md

```diff
@@ -34,6 +34,9 @@ The app should now be live at `APPNAME.18f.gov`.
 
 In general, applications will be easiest to deploy to Cloud Foundry if they follow the [Twelve Factor App](http://12factor.net/) guidelines.
 
+## Setting Environment Variables
+See Cloud Foundry's [documentation on environment variables](http://docs.run.pivotal.io/devguide/deploy-apps/environment-variable.html).
```

Comment:
Mind using the community docs instead?

https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html

_Starting to think we should add an automated test for this..._